src/Altinn.App.Core/Models/DataElementIdentifier.cs

```diff
@@ -37,17 +42,27 @@ public DataElementIdentifier(Guid guid)
         Id = guid.ToString();
     }
 
+    /// <summary>
+    /// Constructor that initializes from a <see cref="DataElement"/>
+    /// </summary>
+    public DataElementIdentifier(DataElement dataElement)
+    {
+        Guid = Guid.Parse(dataElement.Id);
+        Id = dataElement.Id;
+        DataTypeId = dataElement.DataType;
```

Comment:
Why would we name it `DataTypeId` here as opposed to `DataType`?

Reply:
Because I get confused whether something is the dataType object or the ID. I can't change `dataElement.DataType` to `dataElement.DataTypeId`. Do you think I should follow the naming in storage?

Reply:
I see that we use `DataTypeId` a couple of more places (signing and process stuff), so I'm fine with it